match/lib/match/encrypt.rb

```diff
@@ -93,7 +93,9 @@ def crypt(path: nil, password: nil, encrypt: true)
       command << "-out #{tmpfile.shellescape}"
       command << "-a"
       command << "-d" unless encrypt
-      command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show an error message if something goes wrong
+      unless Helper.is_windows?
+        command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show an error message if something goes wrong
```

Comment:
I believe you want `"2> nul"` in Windows to achieve the same thing

Reply:
Thanks, will check that out.

Reply:
Googling confirmed, so I implemented this. 
(Hope I got the `unless`es right...)